### PR TITLE
restructured "starting an own OSS project"

### PR DESCRIPTION
**Pull request by misappi**
* restructured the "starting an OSS project" part
* aligned toc and the content markdown file
* added chapter about the FSFE REUSE Tool

### CODE REVIEW

File: toc.md [hunk 1]
```diff
@@ -24,36 +23,42 @@
...
+  * Active or Development Phase
+  * Mature or Maintenance Phase
+  * Obsolete or End of Life Phase
+
+* Legal and governance considerations
+  * Which license to select
+  * Contributor License Agreement (CLA), Developer Certificate of Origin (DCO)
+
+* Community Management
+  * Governance (open governance, transparency about governance)
```
Comment (cornelius) [inline toc.md:48]: I would see governance as something which goes beyond community, so I wouldn't put it under community management. I tend to think that keeping it under the title "legal and governance considerations" in the order as it is in the document (license, cla, code of conduct, governance, community management) would work ok. This goes from most formal to least formal, covering the different aspects.

File: starting-oss-projects.md [hunk 2]
```diff
@@ -68,6 +68,8 @@ In addition, the following aspects should be considered in the planning phase:
 
 #### Different Project Levels
 
+TBD: Move out of "project lifecycle" chapter?
```
Comment (cornelius) [inline starting-oss-projects.md:71]: I agree that it feels a bit our of place here. Maybe it would fit into the governance section?

Comment (misappi) [reply starting-oss-projects.md:71]: Yes, that would fit. I will change it.

File: toc.md [hunk 3]
```diff
@@ -24,36 +23,42 @@
...
-    * Setting up a repo (templates, health files, license, readme, contribution guidelines)
-    * REUSE
-    * CLA Assistant
-    * Credential scanner
-    * CII Badge
-    * Day-2-ops (CI/CD, security, license scanning, …)
-
-  * Build an open source metrics strategy when releasing to open source projects
-     * Goal-question-metric approach
+* Not to start an OS project?
```
[the remark this answers is not in the document]
Comment (misappi) [reply toc.md:35]: Yes, you are right. I will change it.

File: toc.md (hunk 1 above)
Comment (misappi) [reply toc.md:48]: I was indeed a little bit unsure where to put it. As you suggested I will put it back to the "legal and governance ..." chapter.

**Pull request merged**